pulp_rpm/app/migrations/0026_checksum_null.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2020-10-09 14:58
+
+from django.db import migrations, models
+from django.db.migrations import exceptions
+
+
+def check_checksum(apps, schema_editor):
+    checksum_model = apps.get_model("rpm", "Checksum")
+    q_bad_dist_tree = checksum_model.objects.filter(checksum=None)
+    if q_bad_dist_tree:
+        dist_tree_pks = {
+            c.distribution_tree.pk for c in checksum_model.objects.filter(checksum=None)
+        }
+        raise exceptions.AmbiguityError(
```

Comment:
It's an interesting find, I see that you are trying to figure out the most appropriate exception for this.
I think the `AmbiguityError` has a specific purpose, see [the docstring for it](https://github.com/django/django/blob/e18156b6c35908f2a4026287b5225a6a4da8af1a/django/db/migrations/exceptions.py#L4).

I think the error here is quite application specific, so I'm unsure about using django exceptions. How about just plain old `ValueError` ? @ppicka, @ggainey , wdyt?

Reply:
In [that](https://github.com/django/django/blob/e18156b6c35908f2a4026287b5225a6a4da8af1a/django/db/migrations/exceptions.py#L4) case the `ValueError` seems the best.